project.go

```diff
@@ -65,8 +65,8 @@ var projectJSONToColumns = map[string]string{
 // @description while the matching filters are meant for searches by humans where it tries to find soft matches and is therefore inaccurate by nature.
 // @tags project
 // @param orderby query []string false "Sorting orders. Takes the property name followed by either 'asc' or 'desc'. Can be specified multiple times for more granular sorting. Defaults to `?orderby=projectId desc`"
-// @param limit query int false "Number of results to return. No limit if unset or non-positive. Required if `offset` is used."
-// @param offset query int false "Skipped results, where 0 means from the start." minimum(0)
+// @param limit query int false "Number of results to return. No limit if unset or non-positive. Required if `offset` is used." default(100)
```

Comment:
Dupe of previous.

Description here doesn't seem to match the changelog description:
> By default all these new endpoints use a default limit of 100, but this can
  be disabled by specifying `?limit=0`.

Assuming the changelog description is the actual behavior?
```suggestion
// @param limit query int false "Number of results to return. Set to 0 or non-positive to disable. Required if `offset` is used." default(100)
```
Alt.
```suggestion
// @param limit query int false "Number of results to return. Set to 0 to disable. Required if `offset` is used." minimum(0) default(100)
```